yb-voyager/src/issue/constants.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package issue
+
+// Types
+const (
+	ADVISORY_LOCKS = "advisory_locks"
+	SYSTEM_COLUMNS = "system_columns"
+	XML_FUNCTIONS  = "xml_functions"
+)
+
+// Object types
+const (
+	TABLE_OBJECT_TYPE     = "table"
```

Comment:
value should be all caps as we use it currently like that - `TABLE`, `FUNCTION`.. so similar for `DML QUERY`

Reply:
Should we do this for even the types ? (ADVISORY_LOCKS, SYSTEM_COLUMNS) ?

Reply:
yeah should be fine